packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -222,19 +202,59 @@ export class FrontendPluginImpl {
       );
     }
 
-    if (isArmSupportEnabled()) {
-      const config = await FrontendConfig.fromPluginContext(ctx);
-      config.endpoint = getArmOutput(ctx, ArmOutput.FrontendEndpoint) as string;
-      config.domain = getArmOutput(ctx, ArmOutput.FrontendDomain) as string;
-      config.syncToPluginContext(ctx);
+    return ok(undefined);
+  }
 
-      const client = new AzureStorageClient(config);
-      await runWithErrorCatchAndThrow(
-        new EnableStaticWebsiteError(),
-        async () => await client.enableStaticWebsite()
+  public async postProvisionWithArm(ctx: PluginContext): Promise<TeamsFxResult> {
```

Comment:
NOT suggest to copy the same logic function into two functions since it will highly increase maintenance effort. 

Suggest to extract the get-config logic as a sub-function so we can move the isArmSupportEnabled  judgement to the sub-function to make the logic more clear.

Reply:
As for frontend part. Since I'm going to update the whole post provision method, it's ok to keep this and I will do some refactor before my changes.